tests/Integration/Controller/SecurityControllerTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace App\Tests\Integration\Controller;
+
+use App\DataFixtures\ScorerFixture;
+use App\Tests\BaseWebTestCase;
+use Symfony\Component\Security\Core\Security;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+class SecurityControllerTest extends BaseWebTestCase
+{
+    public function setUp()
+    {
+        parent::setUp();
+        $this->addFixture(new ScorerFixture());
+        $this->loadFixtures();
+    }
+
+    public function testLoginWithLastUserName()
```

Comment:
Test is failing: `ArgumentCountError: Too few arguments to function App\DataFixtures\ScorerFixture::__construct(), 0 passed in /home/gyula/Projects/human_scoring/tests/Integration/Controller/SecurityControllerTest.php on line 14 and exactly 1 expected`